thesis-gui/src/components/auth/sign-up/SignUp.tsx

```diff
@@ -77,11 +79,9 @@ export const SignUp: React.FC = () => {
 
     return (
         <IonPage>
-            <PageHeader/>
+            <PageHeader name={ PAGE_NAME } condense={ false } />
             <IonContent className="ion-padding">
-                <IonToolbar>
-                    <IonTitle size="large">Sign Up</IonTitle>
-                </IonToolbar>
+                <PageHeader name={ PAGE_NAME } condense={ true } />
```

Comment:
Same as above. I cannot see this nested header displayed on the page. What does it look like on your computer?

Reply:
Same comment for other components.